Packs/Office365AndAzureAuditLog/Integrations/MicrosoftPolicyAndComplianceAuditLog/README.md

```diff
@@ -21,6 +21,13 @@ Use the integration to get logs from the O365 service.
     3. Run the ***o365-auditlog-auth-test*** command to verify that the authorization process was implemented correctly.
 
 
+## Required Permissions To Search Audit Logs
+- Minimum Required Exchange roles are **Audit Logs** or **View-Only Audit Logs**
+- Go to [The Microsoft Admin Portal](https://admin.microsoft.com/Adminportal#/homepage)
+- Click **Show All** --> **Roles** --> **Roles Assignments** --> **Exchange section**.
+- Click **Add role group** --> Choose for name and description --> Choose Audit Logs or View-Only Audit Logs Roles --> Choose the members --> Add role group
```

Comment:
> Choose for name and description

I'm not familiar with the context so perhaps this happens to be correct, but unless this phrasing is from the source materials then I would change it to `Choose the name and description`.

Reply:
So which is it?

Reply:
The phrasing here is not part of the source materials.